selfdrive/test/process_replay/compare_logs.py

```diff
@@ -58,7 +58,7 @@ def compare_logs(log1, log2, ignore_fields=None, ignore_msgs=None, tolerance=Non
   if ignore_msgs is None:
     ignore_msgs = []
 
-  log1, log2 = [list(filter(lambda m: m.which() not in ignore_msgs, log)) for log in (log1, log2)]
+  log1, log2 = (list(filter(lambda m: m.which() not in ignore_msgs, log)) for log in (log1, log2))
```

Comment:
I think this should also work.
```suggestion
  log1, log2 = ([filter(lambda m: m.which() not in ignore_msgs, log)] for log in (log1, log2))
```

Reply:
Btw squashing is done automatically on merge, don't think it's needed in a PR

Reply:
Sorry, though that [] and list() were interchangeable, apparently not in this case